kernel-rs/src/kernel.rs

```diff
@@ -61,7 +60,8 @@ pub struct Kernel {
     ///
     /// This is a global instead of allocated because it must be multiple contiguous pages, which
     /// `kernel().alloc()` doesn't support, and page aligned.
-    // TODO(efenniht): I moved out pages from Disk. Did I changed semantics (pointer indirection?)
+    // TODO(https://github.com/kaist-cp/rv6/issues/373)
+    // I moved out pages from Disk. Did I changed semantics (pointer indirection?)
```

Comment:
#355 가 merge되면서 `virtqueue` 필드가 아예 필요 없어졌습니다. 제가 PR에서 지웠다고 생각했는데 지금 보니까 안 지웠네요.. 죄송합니다.

아래처럼 바뀌어야 합니다. 이 PR에서 같이 수정해 주시면 좋을 것 같습니다. :)

```diff
--- a/kernel-rs/src/kernel.rs
+++ b/kernel-rs/src/kernel.rs
@@ -8,7 +8,7 @@ use crate::{
     file::{Devsw, FileTable},
     fs::{FileSystem, Itable},
     kalloc::Kmem,
-    page::{Page, RawPage},
+    page::Page,
     param::{NCPU, NDEV},
     plic::{plicinit, plicinithart},
     println,
@@ -56,13 +56,6 @@ pub struct Kernel {

     pub bcache: Bcache,

-    /// Memory for virtio descriptors `&c` for queue 0.
-    ///
-    /// This is a global instead of allocated because it must be multiple contiguous pages, which
-    /// `kernel().alloc()` doesn't support, and page aligned.
-    // TODO(efenniht): I moved out pages from Disk. Did I changed semantics (pointer indirection?)
-    virtqueue: [RawPage; 2],
-
     pub devsw: [Devsw; NDEV],

     pub ftable: FileTable,
@@ -85,7 +78,6 @@ impl Kernel {
             procs: ProcessSystem::zero(),
             cpus: [Cpu::new(); NCPU],
             bcache: Bcache::zero(),
-            virtqueue: [RawPage::DEFAULT, RawPage::DEFAULT],
             devsw: [Devsw {
                 read: None,
                 write: None,
```